### ISSUE TITLE
Github Sponsors etc.

### ISSUE THREAD

**Issue by JoshFerge** [tfranzel/drf-spectacular#1047]
Hello! does this project have a way to sponsor it? at @getsentry we would love to sponsor this project as part of our open source initiatives!

**Comment by jbbqqf** [comment]
Hi! Triaging older issues — I think this one is out of scope for the issue tracker and can be closed.

Evidence:
- Configuring sponsor buttons / funding links is set up by the repo owner via a `.github/FUNDING.yml` file ([GitHub docs](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/displaying-a-sponsor-button-in-your-repository)). I don't see one in the repo today (`ls .github/` only shows `ISSUE_TEMPLATE/` and `workflows/`), so this is purely an owner-side decision that an issue can't unblock.
- There's also no public discussion from @tfranzel on this in the issue tracker that I can find — the right channel for a sponsorship offer like Sentry's is probably a direct email to the maintainer (the address is on the `pyproject.toml` `[project.authors]` entry).

If the maintainer wants to enable Sponsors, that's their call regardless of this thread. Either way, this issue doesn't seem to be tracking actionable repo work — would you mind closing it out?

---
*Disclosure: I drafted this comment with help from Claude Code while triaging stale issues; the references above were verified manually.*
